package.json

```diff
@@ -2,78 +2,82 @@
   "name": "ember-cli-conditional-compile",
   "version": "1.2.0",
   "description": "Conditional compilation (feature-flags) for Ember apps",
+  "keywords": [
+    "ember-addon",
+    "ember-cli",
+    "feature-flags",
+    "ember-compile"
+  ],
+  "repository": "https://github.com/minichate/ember-cli-conditional-compile.git",
+  "license": "MIT",
+  "author": "",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
   "scripts": {
+    "build": "ember build",
     "lint": "eslint app config lib tests",
     "start": "ember server",
-    "build": "ember build",
     "test": "yarn test:node && yarn test:ember",
-    "test:node": "mocha node-tests",
-    "test:ember": "ember test"
+    "test:ember": "ember test",
+    "test:node": "mocha node-tests"
   },
-  "repository": "https://github.com/minichate/ember-cli-conditional-compile.git",
-  "engines": {
-    "node": "12.* || 14.* || >= 16"
-  },
-  "ember": {
-    "edition": "classic"
+  "dependencies": {
+    "babel-plugin-inline-replace-variables": "1.3.1",
+    "broccoli-babel-transpiler": "^7.1.1",
+    "broccoli-funnel": "~2.0.1",
+    "broccoli-replace": "0.12.0",
+    "chalk": "~2.4.2",
+    "ember-cli-babel": "^7.26.11",
```

Comment:
Would this PR still work if you left `ember-cli-babel` at `~7.4.0`? Any bump to `ember-cli-babel` would be considered a breaking change and I should probably release it as a new major version. If possible, it would be simpler to keep it at 7.4.0 for now

Reply:
Good point, let's just leave it as `^7.26.11`. I'll take another look at this PR later today and merge it in. Thanks!